packages/wallet/src/unipig-transitioner.ts

```diff
@@ -75,7 +77,8 @@ export class UnipigTransitioner extends DefaultWallet {
     stateSolver: RollupStateSolver,
     rollupClient: RollupClient,
     signatureVerifier: SignatureVerifier = DefaultSignatureVerifier.instance(),
-    signatureProvider?: SignatureProvider
+    signatureProvider?: SignatureProvider,
+    wallet?: ethers.Wallet
```

Comment:
I think we should add `getAddress(...)` to the `SignatureProvider` interface. I've run into this in other places as well. It makes sense that we'll need it pretty much everywhere we will use `SignatureProvider`.